monkey/infection_monkey/exploit/wmiexec.py

```diff
@@ -113,33 +133,30 @@ def _exploit_host(self) -> ExploiterResultData:
                 } + build_monkey_commandline(self.servers, self.current_depth + 1)
 
             # execute the remote monkey
+            propagation_timestamp = time()
```

Comment:
As discussed in our daily sync, propagation begins when the file is copied.

Reply:
Changed